src/_pytest/pathlib.py

```diff
@@ -484,73 +484,86 @@ class ImportPathMismatchError(ImportError):
 
 
 def import_path(
-    p: Union[str, "os.PathLike[str]"],
+    path: Union[str, "os.PathLike[str]", Path],
     *,
     mode: Union[str, ImportMode] = ImportMode.prepend,
     root: Path,
+    consider_namespace_packages: bool,
```

Comment:
Why not use a default here?

I prefer to force everywhere that calls it to pass the new option explicitly, lest we miss passing the option in production code by mistake.